scripts/tcpdirect_misc/tcpdirect.spec

```diff
@@ -13,6 +20,8 @@ Summary:        TCPDirect
 License:        MIT AND BSD-3-Clause AND LGPL-3.0
 URL:            https://github.com/Xilinx-CNS/tcpdirect
 Source0:        tcpdirect-%{pkgversion}.tgz
+BuildRequires:  gcc make
+BuildRoot:      %{_builddir}/%{name}-root
```

Comment:
Why did you add `BuildRoot`? This is forbidden in Fedora since EL6.

Reply:
It must be a remnant of some previous work. Fixed in https://github.com/Xilinx-CNS/tcpdirect/pull/31.